lib/ronin/vulns/web_vuln.rb

```diff
@@ -455,7 +455,12 @@ def self.scan(url, query_params:  nil,
 
           if form_params
             vulns.concat(
-              scan_form_params(url,form_params, http: http, **kwargs,&block)
+              case form_params
+              when true
```

Comment:
Also going to need new specs for when `.scan` is given `form_params: true` and `form_data:` is given.